tests/reporting/test_universe.py

```diff
@@ -288,6 +288,62 @@ def test_universe_reporting_fee_divisor(localFixture, chain, populatedUniverse,
 
     assert populatedUniverse.getOrCacheReportingFeeDivisor() == defaultValue / 5 * multiplier
 
+''' TODO
```

Comment:
What is the reason for this dangling TODO?

Reply:
Like why hasn't it been done or why do we still want to do it? There were unit tests for the reporting window previously that could in theory be recreated now for the universe. I delayed that in this PR but its still something that could be done.

Do you think there's value in doing it still?

Reply:
I mean this TODO doesn't have a description and comments out what looks like a complete test.  Does this test fail right now?  Is it because the code is bad or the test is bad?  It sounds like the code is bad based on your description, do we believe that the code paths covered by this test are covered elsewhere (in which case we can delete this test) or do you believe that the code is currently untested, and fixing up this test would resolve that (in which case we should do that).

Reply:
Ah sorry. This is a unit test of the old reporting window market creation code which I moved here and commented out in it's entirety since that function no longer exists but one does for universe.

The universe market creation code is well tested but we could convert this outdated unit test into an equivalent on for the universe if desired. My thought was we would do that at some point.